src/event/event_epoll.c

```diff
@@ -194,7 +202,7 @@ _dispatch_unote_register(dispatch_unote_t du, dispatch_priority_t pri)
 	case DISPATCH_EVFILT_CUSTOM_ADD:
 	case DISPATCH_EVFILT_CUSTOM_OR:
 	case DISPATCH_EVFILT_CUSTOM_REPLACE:
-		du._du->du_wlh = DISPATCH_WLH_GLOBAL;
+		du._du->du_wlh = wlh;
```

Comment:
no it needs to be GLOBAL here, event_kevent.c does the same. it is meant to be that.